curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/670ea2c4f42def724c77e3c7.md

```diff
@@ -0,0 +1,215 @@
+---
+id: 670ea2c4f42def724c77e3c7
+title: Step 38
+challengeType: 0
+dashedName: step-38
+---
+
+# --description--
+
+When you log the value of `entryDropdown.value`, it will output the currently selected value from the dropdown list. 
+
+Initially, this will log `breakfast`, but as the user interacts with the dropdown, it will log the value of the currently selected option.
+
+
```

Comment:
Please review this:

Now you need to log the value of ` entryDropdown.value` , it will output the currently selected value from the dropdown list. Here's how it works:

The `<select>` element has multiple `<option>` elements, each with a value attribute (like "breakfast", "lunch", "dinner" etc.).
By default, the selected option is "Breakfast" because it has the selected attribute.
If you log entryDropdown.value, it will return the value of the currently selected <option>

Initially, this will log "breakfast", but as the user interacts with the dropdown, it will log the value of the currently selected option.

Reply:
I'll do that, first I want to confirm is this correct or any changes needed?

Reply:
Please add it to the PR, it is easier to review